<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QDialog" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelback">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1561</width>
     <height>1026</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/bg.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelnum">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>-30</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>988</x>
     <y>-30</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="reddrum_1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reddrum_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>630</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reddrum_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>730</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reddrum_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>810</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reddrum_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>910</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluedrum_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluedrum_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>640</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluedrum_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>700</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluedrum_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>810</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluedrum_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>920</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcircle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelgood">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelbad">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelperfect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelscorenum">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>90</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>988</x>
     <y>90</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelbackbot">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>1551</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelbackbot_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>1551</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>labelback</zorder>
  <zorder>labelcircle</zorder>
  <zorder>labelnum</zorder>
  <zorder>label_3</zorder>
  <zorder>reddrum_3</zorder>
  <zorder>reddrum_1</zorder>
  <zorder>bluedrum_3</zorder>
  <zorder>reddrum_5</zorder>
  <zorder>bluedrum_1</zorder>
  <zorder>reddrum_4</zorder>
  <zorder>bluedrum_4</zorder>
  <zorder>reddrum_2</zorder>
  <zorder>bluedrum_5</zorder>
  <zorder>bluedrum_2</zorder>
  <zorder>labelgood</zorder>
  <zorder>labelbad</zorder>
  <zorder>labelperfect</zorder>
  <zorder>labelscorenum</zorder>
  <zorder>label_4</zorder>
  <zorder>labelbackbot</zorder>
  <zorder>labelbackbot_2</zorder>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
